treeherder/perf/management/commands/import_perf_data.py

```diff
@@ -197,8 +198,14 @@ def save_local_data(self):
             print('Saving {0} data...'.format(table_name))
             model_values = properties['model'].objects.using(self.source).filter(
                 pk__in=self.models_instances[table_name])
+            self._ignore_sensitive_fields(table_name, model_values)
             properties['model'].objects.using(self.target).bulk_create(model_values)
 
+    def _ignore_sensitive_fields(self, table_name, model_values):
+        if table_name == 'performance_alert':
+            for model in model_values:
+                model.classifier = None
```

Comment:
Why setting classifier to None ignores some fields?

Reply:
> Why setting classifier to None ignores some fields?

The UPSTREAM_DATABASE_URL credentials I'm using don't have enough privilege to read the `auth_user` table. Thus, trying to set the `classifier` field ends up in a runtime error.